src/prelude/internals.mo

```diff
@@ -481,26 +486,26 @@ func @timer_helper() : async () {
         case null ();
         case (?n) {
             gatherExpired(n.ante);
-            if (n.expire > 0 and n.expire <= now and gathered < thunks.size()) {
+            if (n.expire[0] > 0 and n.expire[0] <= now and gathered < thunks.size()) {
                 thunks[gathered] := ?(n.job);
                 switch (n.delay) {
                     case (?delay) {
                         // re-add the node
-                        let expire = n.expire + delay;
+                        let expire = n.expire[0] + delay;
                         // N.B. insert only works on pruned nodes
                         func reinsert(m : ?@Node) : @Node = switch m {
-                            case null ({ n with var expire; ante = null; dopo = null });
```

Comment:
So is this the operation that was problematic? What is particular about aliased `var` fields that is a problem?